README.md

```diff
@@ -20,6 +20,10 @@ The current Swift Server work group consists of the following people:
 
 With Tom Doron ([@tomerd](https://github.com/tomerd), Apple) representing the Swift core team.
 
+## Security
+
+Please refer to our [Security](security.md) page for anything related to security of any software related to the SSWG or listed on the SSWG's package index.
```

Comment:
Tweaked some grammar to avoid repetition and slightly increase formality... not sure if it's worth it though.
```suggestion
Please refer to our [Security](security.md) page for matters related to security of any software affiliated with the SSWG or listed on the SSWG's package index.
```